website/views.py

```diff
@@ -133,57 +131,17 @@ def index():
     try:  # Check if we're on an institution landing page
         #TODO : make this way more robust
         institution = Institution.objects.get(domains__contains=[request.host.lower()], is_deleted=False)
-        inst_dict = serialize_institution(institution)
-        inst_dict.update({
-            'home': False,
-            'institution': True,
-            'redirect_url': '{}institutions/{}/'.format(DOMAIN, institution._id),
-        })
-
-        return inst_dict
+        return redirect('{}institutions/{}/'.format(DOMAIN, institution._id))
     except Institution.DoesNotExist:
-        pass
-
-    return home()
-
-@ember_flag_is_active('ember_home_page')
-def home():
-    user_id = get_current_user_id()
-    if user_id:  # Logged in: return either landing page or user home page
-        all_institutions = (
-            Institution.objects.filter(
-                is_deleted=False,
-                nodes__is_public=True,
-                nodes__is_deleted=False,
-                nodes__type='osf.node'
-            )
-            .annotate(Count('nodes'))
-            .filter(nodes__count__gte=INSTITUTION_DISPLAY_NODE_THRESHOLD)
-            .order_by('name').only('_id', 'name', 'logo_name')
-        )
-        dashboard_institutions = [
-            {'id': inst._id, 'name': inst.name, 'logo_path': inst.logo_path_rounded_corners}
-            for inst in all_institutions
-        ]
-
-        return {
-            'home': True,
-            'dashboard_institutions': dashboard_institutions,
-        }
-    else:  # Logged out: return landing page
-        return {
-            'home': True,
-        }
-
+        return use_ember_app()
```

Comment:
Simplest solution to strip out all this legacy code seemed to be to leave the routes and views in place, but calling `use_ember_app` after any necessary view rerouting logic.